lte/gateway/docker/services/python/Dockerfile

```diff
@@ -126,6 +127,7 @@ RUN apt-get update && apt-get install -y \
   python3-eventlet \
   python3-pystemd \
   python3-jinja2 \
+  python3-scapy \
```

Comment:
In other places scapy has a lower bound pinning - is this needed here too?

Reply:
As discussed in person, the versions of scapy in Ubuntu 20 and 22 seem to be fine.